sdks/python/apache_beam/typehints/typecheck.py

```diff
@@ -82,7 +85,7 @@ def wrapper(self, method, args, kwargs):
     except TypeCheckError as e:
       error_msg = ('Runtime type violation detected within ParDo(%s): '
                    '%s' % (self.full_label, e))
-      raise TypeCheckError, error_msg, sys.exc_info()[2]
+      raise_with_traceback(TypeCheckError(error_msg), sys.exc_info()[2])
```

Comment:
I mean we can, but since were constructing a new one this is maybe nicer? `raise_` is a little ugly imho. But I'm happy to use it more places if we need to

Reply:
+1 to raise_with_traceback.